otg.go

```diff
@@ -40,15 +42,15 @@ func (o *OTG) NewConfig() gosnappi.Config {
 }
 
 // PushConfig pushes config to the ATE.
-func (o *OTG) PushConfig(t testing.TB, cfg gosnappi.Config) {
+func (o *OTG) PushConfig(t testing.TB, ate *ATEDevice, cfg gosnappi.Config) {
 	t.Helper()
-	const locFormat = "service-ate%d.ceos-simple.svc.cluster.local:5555+service-ate%d.ceos-simple.svc.cluster.local:50071"
 	for _, p := range cfg.Ports().Items() {
-		portNum, err := strconv.Atoi(p.Name()[1:])
-		if err != nil {
-			t.Fatal(err)
-		}
-		p.SetLocation(fmt.Sprintf(locFormat, portNum, portNum))
+		portName := p.Name()
+		portLocation := ate.Port(t, portName).Name()
```

Comment:
Port.Name() will return the name of the interface in kne, e.g. "eth1". Is the portLocation still expected to be a long string like "service-ate1.ceos-simple.svc.cluster.local:5555+service-ate1.ceos-simple.svc.cluster.local:50071"? If it is, could you tell me again how to derive what this long format string should be?

Reply:
@greg-dennis This is an old PR, could you please 
Refer the new PR https://github.com/openconfig/ondatra/pull/10

 From now onwards we don't need the port location to be long format string, only interface name is fine.

This PR was raised long back should I close this PR otherwise it will create confusions.